apps/profile/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from . import views
+
```

Comment:
Bitte hier noch die Variable "app_name='profile'" setzen. Das dient dann dazu, die URLs rückwerts aufzulösen. Zb. kann man dann in einem Template {% url 'profile:index' %} nutzen, um die URL aufbauen zu lassen